core/src/main/java/org/apache/calcite/adapter/enumerable/RexToLixTranslator.java

```diff
@@ -376,6 +394,17 @@ private Expression converterToTime(RelDataType sourceType, Expression operand) {
           BuiltInMethod.TIMESTAMP_WITH_LOCAL_TIME_ZONE_TO_TIME.method,
           operand,
           Expressions.call(BuiltInMethod.TIME_ZONE.method, root)));
+      break;
+    // SQL 2011 Part2 Section 4.6 General Rules 15.e
+    case TIMESTAMP_WITH_TIME_ZONE:
```

Comment:
implement a `timestampWithoutTimeZoneToTimeWithoutTimeZone` and call it here?

Reply:
ditto